tests/schemas/valid/test_mobile_number.json

```diff
@@ -0,0 +1,58 @@
+{
+  "mime_type": "application/json/ons/eq",
+  "language": "en",
+  "schema_version": "0.0.1",
+  "data_version": "0.0.3",
+  "survey_id": "001",
+  "title": "Other input fields",
```

Comment:
I know this is only in validator but name should be relevant. Also, we should try to avoid unnecessary properties in these schemas like `q_code` which is only relevant to data version 0.0.1 anyway.

Reply:
Sure, these have been copied from another schema. Would be good if we had some sort of authoring tool for this 🤔

Reply:
Ya, I think is an issue in a lot of schemas unfortunately.